controllers/projectreference/projectreference_adapter.go

```diff
@@ -465,8 +465,13 @@ func (r *ReferenceAdapter) deleteServiceAccount() error {
 
 		return operrors.Wrap(err, "could not get the SA, something happened")
 	}
+
 	r.logger.V(1).Info("after get")
 
+	if err := r.DeleteIAMPolicy(sa.Email, 0); err != nil {
```

Comment:
**nitpick:** use the text format instead of the numerical struct value.  
```suggestion
	if err := r.DeleteIAMPolicy(sa.Email, util.ServiceAccount); err != nil {
```

Reply:
Thank you!